src/App.js

```diff
@@ -1,20 +1,14 @@
-import axios from "axios";
-import { useState } from "react";
 import { Route, Routes } from "react-router-dom";
-import "./App.css";
 import HomePage from "./components/Homepage/HomePage";
 import MoviePage from "./components/Moviepage/MoviePage";
 
 function App() {
-  const [movieInfo, setMovieInfo] = useState([]);
-
   return (
-    <div className="App">
-      <Routes>
-        <Route path="/" element={<HomePage setMovieInfo={setMovieInfo} />} />
-        <Route path="movie/:id" element={<MoviePage movieInfo={movieInfo} />} />
-      </Routes>
-    </div>
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="/page/:page/movie/:id" element={<MoviePage />} />
+      <Route path="/page/:page" element={<HomePage />}></Route>
```

Comment:
Also seems reasonable to unite path="/" and path="/page/:page" into one path: path="/:page"

Reply:
fixed